.circleci/config.yml

```diff
@@ -202,6 +203,34 @@ jobs:
       - slack/status:
           fail_only: true
           mentions: "channel"
+
+  validate_radiation_port:
+    machine:
+      image: ubuntu-2004:202111-02
+    environment:
+      GOOGLE_PROJECT_ID: vcm-ml
+      GOOGLE_APPLICATION_CREDENTIALS: /tmp/key.json
+    steps:
+      - checkout
+      - gcp-cli/install
```

Comment:
It's probably more elegant to manage all dependencies with nix. This will ensure the CI environment remains as close as possible to the dev environment.  

Currently, the shell.nix doesn't include gsutil, but it probably should. You can do this by adding 
```
pkgs.google-cloud-sdk
```
near the other deps in `shell.nix`.

99% of things we would want to install are available in nix. you can search for them here: https://search.nixos.org/packages?channel=22.05&show=google-cloud-sdk&from=0&size=50&sort=relevance&type=packages&query=gsutil